src/Login/Login.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import Card from '@material-ui/core/Card';
+import { Link } from 'react-router-dom';
+
+class Login extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      username: '',
+      password: '',
+    };
+
+    this.handleInputChange = this.handleInputChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
```

Comment:
Most of this code won't be needed anymore after switching to functional components, you would create local state like this:
```
const [username, setUsername] = useState('');
const [password, setPassword] = useState('');
```